code/modules/vehicles/vehicle.dm

```diff
@@ -0,0 +1,138 @@
+
+/obj/vehicle
+	name = "vehicle"
+	desc = "A basic vehicle, vroom"
+	icon = 'icons/obj/vehicles.dmi'
+	icon_state = "fuckyou"
```

Comment:
well it serves it's purpose, it makes /obj/vehicle invisible by default, which further explains that you shouldn't create instances of the base class, allowing admins/coders alike to see that it is an abstract class.

so yes it does actually do it's job, but its fun to have it say fuckyou